tests/raftstore/test_split_region.rs

```diff
@@ -101,28 +106,56 @@ fn test_base_split_region<T: Simulator>(cluster: &mut Cluster<T>, right_derive:
 fn test_node_base_split_region_left_derive() {
     let count = 5;
     let mut cluster = new_node_cluster(0, count);
-    test_base_split_region(&mut cluster, false);
+    test_base_split_region(&mut cluster, Cluster::must_split, false);
 }
 
 #[test]
 fn test_node_base_split_region_right_derive() {
     let count = 5;
     let mut cluster = new_node_cluster(0, count);
-    test_base_split_region(&mut cluster, true);
+    test_base_split_region(&mut cluster, Cluster::must_split, true);
 }
 
 #[test]
 fn test_server_base_split_region_left_derive() {
     let count = 5;
     let mut cluster = new_server_cluster(0, count);
-    test_base_split_region(&mut cluster, false);
+    test_base_split_region(&mut cluster, Cluster::must_split, false);
 }
 
 #[test]
 fn test_server_base_split_region_right_derive() {
     let count = 5;
     let mut cluster = new_server_cluster(0, count);
-    test_base_split_region(&mut cluster, true);
+    test_base_split_region(&mut cluster, Cluster::must_split, true);
+}
+
+#[test]
+fn test_node_manual_split_region_right_derive() {
+    let count = 5;
+    let mut cluster = new_node_cluster(0, count);
+    test_base_split_region(&mut cluster, Cluster::must_manual_split, false);
+}
+
+#[test]
+fn test_node_manual_split_region_left_derive() {
+    let count = 5;
+    let mut cluster = new_node_cluster(0, count);
+    test_base_split_region(&mut cluster, Cluster::must_manual_split, true);
+}
+
+#[test]
+fn test_server_manual_split_region_right_derive() {
+    let count = 5;
+    let mut cluster = new_server_cluster(0, count);
+    test_base_split_region(&mut cluster, Cluster::must_manual_split, false);
+}
+
+#[test]
+fn test_server_manual_split_region_left_derive() {
```

Comment:
do we have a test to send multi Split requests with the same split key? Maybe only the first can be executed successfully.

Reply:
any update?